airbyte-integrations/base-python/bin/generate-protocol-files.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+
+YAML_DIR=airbyte-protocol/models/src/main/resources/airbyte_protocol
```

Comment:
any reason not to make this part of the gradle more directly? I think having split build logic in and out of gradle scripts can get confusing.